### PR TITLE
Copyedit CTC Traders API v1 Markdown content

### PR DESCRIPTION
**Pull request by fmcgrath**

### CODE REVIEW

File: resources/public/api/conf/1.0/docs/overview.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 
-Further details of the User Restricted Authentication are given on the
-[Authorisation](https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation) page.
+You can also use the HMRC sandbox environment to run tests for Great Britain and Northern Ireland transit movements.
+
+You should read:
+
+- [Authorisation](https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/user-restricted-endpoints) for information about user-restricted API authentication
+- [Roadmap](https://developer.service.hmrc.gov.uk/roadmaps/common-transit-convention-traders-roadmap/) for information about CTC Traders API releases
+- [Service Guide](https://developer.service.hmrc.gov.uk/guides/common-transit-convention-traders-service-guide/) to learn how to use the CTC Traders API with your software
```
Comment (PeterBandura) [inline resources/public/api/conf/1.0/docs/overview.md:14]: This is a link based on a legacy repo. It works due to a redirect to the new URL. We could take this opportunity to update the link. (https://developer.service.hmrc.gov.uk/guides/ctc-traders-phase4-service-guide/). Also, can we replace the absolute link with a relative one?

File: resources/public/api/conf/1.0/docs/overview.md [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+- [Testing Guide](https://developer.service.hmrc.gov.uk/guides/common-transit-convention-traders-testing-guide/) to check that your software is compatible with the CTC Traders API
```
Comment (PeterBandura) [inline resources/public/api/conf/1.0/docs/overview.md:15]: This is a link based on a legacy repo. It works due to a redirect to the new URL. We could take this opportunity to update the link. (https://developer.service.hmrc.gov.uk/guides/ctc-traders-phase4-testing-guide/). Also, can we replace the absolute link with a relative one?

File: resources/public/api/conf/1.0/docs/pushPull.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-* this new functionality will send you a notification each time there is a new message for you to read
-* for messages less than 100KB the Push Notification will contain the message body
-* the Push Notification will have a field containing the messageURI
+* if your endpoint is hosted by Amazon Web Services (AWS), you must use either [edge-optimised custom domain names](https://docs.aws.amazon.com/apigateway/latest/developerguide/how-to-edge-optimized-custom-domain-name.html) or [regional custom domain names](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-regional-api-custom-domain-create.html)
+* you will receive a push notification each time when there is a new message for you to read
+* for messages less than 100KB, a push notification will contain the message body
+* a push notification will have a field containing the message URI
 * you can then use this URI to download the XML message from the CTC Traders API
```
Comment (PeterBandura) [inline resources/public/api/conf/1.0/docs/pushPull.md:7]: Do we need "then"?

**Review by PeterBandura — DISMISSED** [review]
A couple of minor points: 
1) We could update the defunct links to the Service Guide and Testing Guide (they work due to redirects but it might be better to update the links themselves)
2) Do we need "then" in the final bullet point in the push-pull file.

File: resources/public/api/conf/1.0/docs/pushPull.md (hunk 3 above)
Comment (fmcgrath) [reply resources/public/api/conf/1.0/docs/pushPull.md:7]: Well spotted. Have removed it.

File: resources/public/api/conf/1.0/docs/overview.md (hunk 2 above)
Comment (fmcgrath) [reply resources/public/api/conf/1.0/docs/overview.md:15]: Have updated all these links to be relative. I asked James about the P4 testing guide link. He wants it to changed to the P5 guide but also comment out the bullet point until the P5 testing guide reaches PROD, which is what I have done.

**Review by drnaylor-oc — APPROVED** [review]
That's far better than what we had, thanks!

**Pull request merged**